source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -152,10 +156,38 @@ bool TrySetupContext(string kubeConfig, string @namespace, string accountType)
 
         bool CreateNamespace(string @namespace)
         {
-            if (TryExecuteCommandWithVerboseLoggingOnly("get", "namespace", @namespace))
+            log.Info($"SCME CreateNamespace: {kubectl.GetHashCode()}");
+            log.Info($"SCME CreateNamespace EnvVars: '{JsonConvert.SerializeObject(kubectl.environmentVars)}'");
+            log.Info($"SCME CreateNamespace workingDirectory: '{kubectl.workingDirectory}'");
+
+            // Load from in-cluster configuration:
+            var config = KubernetesClientConfiguration.InClusterConfig();
```

Comment:
I hacked in the use of [KubernetesClient](https://github.com/kubernetes-client/csharp) to see if it would work, and it did! I had a look in the `InClusterConfig()` method and found that it was setting a bunch of things from environment variables so we might be able to do something similar to solve this problem, or we just update calamari to use KubernetesClient?